content/it/docs/concepts/configuration/configmap.md

```diff
@@ -0,0 +1,242 @@
+1. Scrivere il codice da eseguire all'interno del Pod che utilizza l'API di Kubernetes per leggere la ConfigMap
+
+Questi metodologie differenti permettono di utilizzare diversi metodi per modellare i dati che saranno consumati.
+Per i primi tre metodi, il 
+{{< glossary_tooltip text="kubelet" term_id="kubelet" >}} utilizza i dati della
+ConfigMap quando lancia il container (o più) in un Pod.
+
+Per il quarto metodo dovrai scrivere il codice per leggere la ConfigMap e i suoi dati.
+Comunque, poiché stai utilizzando l'API di Kubernetes direttamente, la tua applicazione può
+sottoscriversi per ottenere aggiornamenti ogniqualvolta la ConfigMap cambia, e reagire
+quando ciò accade. Accedendo direttamente all'API di Kubernetes, questa
+tecnica ti permette anche di accedere a una ConfigMap in namespace differenti.
+
+Ecco un esempio di Pod che usa i valori da `game-demo` per configurare il container:
+
+```yaml
+apiVersion: v1
+kind: Pod
+metadata:
+  name: configmap-demo-pod
+spec:
+  containers:
+    - name: demo
+      image: alpine
+      command: ["sleep", "3600"]
+      env:
+        # Definire la variabile d'ambiente
+        - name: PLAYER_INITIAL_LIVES # Notare che il case qui è differente
+                                     # dal nome della key nella ConfigMap.
+          valueFrom:
+            configMapKeyRef:
+              name: game-demo           # La ConfigMap da cui proviene il valore.
+              key: player_initial_lives # La chiave da recuperare.
+        - name: UI_PROPERTIES_FILE_NAME
+          valueFrom:
+            configMapKeyRef:
+              name: game-demo
+              key: ui_properties_file_name
+      volumeMounts:
+      - name: config
+        mountPath: "/config"
+        readOnly: true
+  volumes:
+    # Settare i volumi al livello del Pod, in seguito montarli nei containers all'interno del Pod
+    - name: config
+      configMap:
+        # Fornire il nome della ConfigMap che vuoi montare.
+        name: game-demo
+        # Una lista di chiavi dalla ConfigMap per essere creata come file
+        items:
+        - key: "game.properties"
+          path: "game.properties"
+        - key: "user-interface.properties"
+          path: "user-interface.properties"
+```
+
+Una ConfigMap non differenzia tra le proprietà di una singola linea e un file con più linee e valori.
+L'importante è il modo in cui i Pods e gli altri oggetti consumano questi valori.
+
+Per questo esempio, definire un volume e montarlo all'interno del container `demo` come `/config` crea
+due files,
+`/config/game.properties` e `/config/user-interface.properties`,
+sebbene ci siano quattro chiavi nella ConfigMap. Ciò avviene perché la definizione del Pod
+specifica una lista di `items` nella sezione dei `volumes`.
+Se ometti del tutto la lista degli `items`, ogni chiave nella ConfigMap diventerà
+un file con lo stesso nome della chiave, e otterrai 4 files.
+
+## Usando le ConfigMaps
```

Comment:
```suggestion
## Usare le ConfigMaps
```
It isn't the literary translation but I think it is better in the context
Same below

Reply:
Fixed also below, thanks for time and patience